pages/landing/hipaa/template/index.html

```diff
@@ -0,0 +1,66 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Achieve HIPAA compliant infrastructure, instantly
+excerpt: Go live with a customized, HIPAA-compliant cloud infrastructure and sample application in about a day.
+permalink: /hipaa-cloud-compliance/
+
+hero:
+  image: /assets/img/infrastructure-cube-icons/infrastructure-in-a-day.png
+  cta: Join the Waitlist
```

Comment:
I think what we may want here is to have a CTA like "request early access." The key message to send is:

- We are working on HIPAA.
- Here's all the work it takes: show X items.
- Here are the items we have done: show Y items, where Y is a substantial portion of X.
- Here is what it'll cost when we're done: $A.
- If you sign up early, here are the benefits: only costs $B; get access now and get all of X in 1 day; we'll update you with (X - Y) once it's done; perhaps other incentives such as swag.

Reply:
Initially addressed this in f5869e7f - splitting the single table into two so that the first can answer these questions and highlight the work already accomplished and that still remaining to be done.

Reply:
Button at the very bottom of the page should say "request early access" too.

Reply:
Footer button CTA copy updated to "Request early access" in 4ce0964e